Adds proper motion correction and galaxy support to Contamination tool

hover2pi opened the pull request:
- Corrects RA and Dec of sources in contamination tool using proper motion data from Gaia EDR3.
- Differentiates between stars and galaxies in contamination tool, showing the trace location but not the level of contamination in the galaxy case

pep8speaks commented:
Hello @hover2pi, Thank you for submitting the Pull Request !

* In the file [`exoctk/contam_visibility/field_simulator.py`](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py):

> [Line 41:46](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py#L41): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 44:1](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py#L44): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 46:162](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py#L46): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ':'
> [Line 47:164](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py#L47): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ':'
> [Line 48:165](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py#L48): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ':'
> [Line 58:1](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py#L58): [E302](https://duckduckgo.com/?q=pep8%20E302) expected 2 blank lines, found 1
> [Line 130:34](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py#L130): [E226](https://duckduckgo.com/?q=pep8%20E226) missing whitespace around arithmetic operator
> [Line 185:94](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py#L185): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 450:51](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py#L450): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 493:64](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py#L493): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 494:64](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py#L494): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 495:64](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py#L495): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 585:161](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py#L585): [E241](https://duckduckgo.com/?q=pep8%20E241) multiple spaces after ','
> [Line 812:16](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py#L812): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 822:59](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py#L822): [E226](https://duckduckgo.com/?q=pep8%20E226) missing whitespace around arithmetic operator
> [Line 1100:9](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py#L1100): [E303](https://duckduckgo.com/?q=pep8%20E303) too many blank lines (2)
> [Line 1162:18](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py#L1162): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 1162:18](https://github.com/ExoCTK/exoctk/blob/3ce13dacc13dcbb40a28a21c67bd444aa55b1ab6/exoctk/contam_visibility/field_simulator.py#L1162): [E262](https://duckduckgo.com/?q=pep8%20E262) inline comment should start with '# '

If you have not done so, please consult [PEP8](https://www.python.org/dev/peps/pep-0008/) and [PEP257](https://www.python.org/dev/peps/pep-0257/)

mfixstsci commented:
@hover2pi just looked at the files and noticed that you are adding validation to a form and removing `hotsoss`?

hover2pi commented:
@mfixstsci Yes, I found an error in the Fortney form validation and added a fix to the server error we were getting when someone submits a form with blank fields. Unrelatedly, I removed `hotsoss` as a dependency and moved the relevant code into the contam tool.
